cdap-docs/cdap-apps/source/tracker/index.rst

```diff
@@ -0,0 +1,441 @@
+  :width: 800px
+  :align: center
+  :class: bordered-image
+
+**Audit Log**
+
+The *Audit Log* tab shows each record in the *AuditLog* dataset that has been created for
+that particular entity, displayed in reverse chronological order. Note that due to a
+limitation in CDAP, reading and writing from a flow or service to a dataset shows an
+access of "UNKNOWN" rather than indicating if it was read or write access.
+
+A date menu in the left side of the digram lets you control the time range that the
+diagram displays. By default, the last seven days are used, though a custom range can be
+specified, in addition to common time ranges (two weeks to one year).
+
+.. figure:: ../_images/tracker-audit-log.png
+  :figwidth: 100%
+  :width: 800px
+  :align: center
+  :class: bordered-image
+
+Integrations
+------------
+
+Tracker allows for an easy integration with `Cloudera Navigator
+<https://www.cloudera.com/products/cloudera-navigator.html>`__  by providing a UI to
+connecting to a Navigator instance:
+
+.. figure:: ../_images/tracker-integration-configuration.png
+  :figwidth: 100%
+  :width: 800px
+  :align: center
+  :class: bordered-image
+
+Details on completing this form are described in CDAP's documentation on
+:ref:`Navigator Integration Application <navigator-integration>`.
+
+
+Tracker HTTP RESTful API
+========================
+
+Tracker supports searching of the *AuditLog* dataset through an HTTP RESTful API. To
+search for audit log entries for a particular dataset, stream, or stream view, submit an
+HTTP GET request::
+
+  GET <base-url>/namespaces/<namespace>/apps/_Tracker/services/AuditLog/methods/auditlog/<entity-type>/<name>
+    [?startTime=<time>][&endTime=<time>][&offset=<offset>][&limit=<limit>]
+
+where:
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Parameter
+     - Description
+   * - ``<namespace>``
+     - Namespace ID
+   * - ``<entity-type>``
+     - One of ``dataset``, ``stream``, or ``stream_view``
+   * - ``<name>``
+     - Name of the ``<entity-type>``
+   * - ``<time>`` *(Optional)*
+     - Time range defined by start (*startTime*, default ``0``) and end (*endTime*,
+       default ``now``) times, where the times are either in milliseconds since the start of
+       the Epoch, or a relative time, using ``now`` and times added to it. You can apply
+       simple math, using ``now`` for the current time, ``s`` for seconds, ``m`` for
+       minutes, ``h`` for hours and ``d`` for days. For example: ``now-5d-12h`` is 5 days
+       and 12 hours ago.
+   * - ``<offset>`` *(Optional)*
+     - The offset to start the results at for paging; default is ``0``.
+   * - ``<limit>`` *(Optional)*
+     - The maximum number of results to return in the results; default is ``10``.
+     
+A successful query will return with the results as a field along with a count of the total
+results available, plus the offset used for the set of results returned. This is to allow
+for pagination through the results.
```

Comment:
Should we mention that results are sorted so that the most recent audit event in the time range is returned first?